talks/de.json

```diff
@@ -1,126 +1,151 @@
 		]
 	},
 	"respondDelete": {
 		"default": [
-			"Ok, ich habe den {} für dich gelöscht.",
-			"Erledigt, ich habe deinen {} gelöscht.",
-			"Von dem {} wirst du nichts mehr hören.",
-			"Der {} ist fort. Ok?"
+			"Keine Sorge habe ich gelöscht, dass {} für Sie.",
+			"Alle sortiert, Ihre {} ist wie niemand hier war aufgeräumt.",
+			"Wow, Ich löschte nicht nur, dass {} i es zum Mars auf eine Mission geschickt.",
+			"Scheint tat ich das richtig. Die {} ist jetzt nicht da."
 		],
 		"short": [
-			"Erledigt.",
-			"OK."
+			"Sortiert",
+			"Erledigt"
 		]
 	},
 	"respondFoodTimer": {
 		"default": [
-			"Hey chef, wir sind halb durch den Timer. Zeit nach dem Essen zu schauen!",
-			"Aufgepasst. Du solltest mal schauen wie es mit dem Essen aussieht.",
-			"Riecht das hier verbrannt? Sollte doch gerade mal die hälfte der Zeit sein?",
-			"Ich bekomme langsam hunger, wie weit ist das Essen?"
+			"Hey Chef, wir sind auf halbem Weg durch den Timer. Zeit auf Ihrem Essen zu überprüfen!",
+			"Achtung Koch. Schauen Sie auf Ihr Essen im Falle muss es gerührt oder gedreht.",
+			"Tippen Sie auf, tippen Sie auf. <Pause = „250ms“ /> weiß nicht, ob dieses Mikrofon funktioniert. <Pause = „150ms“ /> Ummmm Achtung Käufer, foodcheck auf Insel 3 bitte",
+			"Muss ich etwas Brandgeruch? Könnte zahlen Scheck auf Ihr Essen gehen jetzt Koch."
 		],
 		"short": [
-			"Schau mal nach dem Essen!",
-			"Essen ist bald fertig."
+			"Überprüfen Sie Ihr Essen!",
+			"Lebensmittel zu überprüfen, Insel 3!"
 		]
 	},
 	"respondToANoReply": {
 		"default": [
-			"ok. Dann nicht.",
-			"Dann habe ich von nichts gehört.",
-			"Nagut, dann schlafe ich jetzt weiter."
+			"ok Ich werde meine Schaltung zurück Bord zu entstauben und vergessen, dass {} ...",
+			"Sicher, ich vergesse gesagt, du auch nur ein Wort ...",
+			"Ok, ich bin beschäftigt Erwärmung Himbeerkuchen sowieso!"
 		],
 		"short": [
-			"ok!",
-			"Klar!"
+			"ok, vergessen!",
+			"Sicher!"
 		]
 	},
 	"respondSetReminder": {
 		"default": [
-			"Erinnerung wurde auf {} gesetzt",
-			"Ich erinner dich um {0} wegen {1}"
+			"Zur Erinnerung eingestellt wurde für {}",
+			"Ich will daran erinnern, auf {0} um die Erinnerung {1}"
 		],
 		"short": [
-			"Erinnerung erstellt.",
-			"Erinnerung bereit."
+			"Erinnerung ist Set",
+			"Sortiert"
 		]
 	},
 	"respondReminderMessage": {
 		"default": [
-			"Welche Nachricht willst du für den {} hinterlegen?",
-			"Wie lautete das Thema für den {}?"
+			"Was Nachricht würden Sie wie ich für diese Einstellung vornehmen, {}",
+			"Was Thema möchten Sie mich diese nennen {}"
 		],
 		"short": [
-			"Wie lautet die Nachricht?"
+			"was ist die Botschaft!",
+			"was ist die {} Nachricht"
 		]
 	},
 	"respondSetDuration": {
 		"default": [
-			"Zu welcher Zeit soll der {} sein?",
-			"Wann soll der {} auftreten?"
+			"Was Zeit wollen Sie das {} Set für?",
+			"Wann wollen Sie die {} wieder auftreten?"
 		],
 		"short": [
-			"Für wann?",
-			"Uhrzeit? Datum?"
+			"Für WHE?",
```

Comment:
that seems like a broken translation: WHE is probably from "when" but was just not translated at all.